README.md

```diff
@@ -49,6 +50,35 @@ Particle orientations are stored as
 [`scipy.spatial.transform.Rotation`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.transform.Rotation.html) objects.
 These transformations should rotate the basis vectors of a reference such that they are correctly oriented in a tomogram.
 
+## `ProjectionModel`
+A `ProjectionModel` is a [pandas `DataFrame`](https://pandas.pydata.org/docs/) with specific column 
+headings for the parameters of a projection model. Together, this information constitues a 'tilt-series alignment'.
+
+| Heading         | Python name   | Semantics                                  |
+|:----------------|:--------------|:-------------------------------------------|
+| `rotation_x`    | ROTATION_X    | tomogram rotation around x-axis            |
+| `rotation_y`    | ROTATION_Y    | tomogram rotation around y-axis            |
+| `rotation_z`    | ROTATION_Z    | tomogram rotation around z-axis            |
+| `dx`            | SHIFT_X       | particle shift in x-dimension              |
+| `dy`            | SHIFT_Y       | particle shift in y-dimension              |
+| `experiment_id` | EXPERIMENT_ID | identifier for micrograph/tilt-series      |
+| `pixel_spacing` | PIXEL_SPACING | isotropic pixel/voxel spacing for shifts   |
+| `source`        | SOURCE        | reference to the file from which data came |
+
+A utility function is also provided for generating projection matrices from these data.
```

Comment:
What transformation are these proj matrices describing exactly?

Reply:
position in tomogram -> position in tilt image, one matrix per tilt image

Reply:
As in, if you apply this to the real object and project along z, you get the tilt image? I would add a small note for this for clarity!

Reply:
yes, if by z you mean the beam axis :)

Reply:
note added